app/models/theme.rb

```diff
@@ -506,6 +525,7 @@ def generate_metadata_hash
 #  color_scheme_id  :integer
 #  remote_theme_id  :integer
 #  component        :boolean          default(FALSE), not null
+#  disabled         :boolean          default(FALSE), not null
```

Comment:
Maybe we go with `enabled` here ... 

Reasoning about `where enabled` is way simpler than thinking about double negatives, `where not disabled`